packages/framework/src/Migrations/Version20230404071649.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Shopsys\MigrationBundle\Component\Doctrine\Migrations\AbstractMigration;
+
+class Version20230404071649 extends AbstractMigration
+{
+    /**
+     * @param \Doctrine\DBAL\Schema\Schema $schema
+     */
+    public function up(Schema $schema): void
+    {
+        $seoRobotsContentSettingCount = $this->sql(
+            'SELECT COUNT(*) FROM setting_values WHERE name = \'seoRobotsContent\' AND domain_id = 1;'
+        )->fetchOne();
+
+        if ($seoRobotsContentSettingCount <= 0) {
+            $this->sql(
+                'INSERT INTO setting_values (name, domain_id, value, type) VALUES (\'seoRobotsContent\', 1, NULL, \'string\')',
```

Comment:
This migration should be done for all domains, not only the first one

Reply:
OK. Looks like I copy it from too much old migration which did not have it.